### PR TITLE
Traktor feature clean up 

### PR DESCRIPTION
**Pull request by daschuer**
I have tweaked the Traktor feature that it is testable on Linux as well.

### CODE REVIEW

File: src/library/traktor/traktorfeature.cpp [hunk 1]
```diff
@@ -302,14 +302,15 @@ void TraktorFeature::parseTrack(QXmlStreamReader &xml, QSqlQuery &query) {
...
+                // which is a folder in /Volumes/ symlinked to root folder /
                 #if defined(__APPLE__)
-                location = "/Volumes/"+volume;
+                location = "/Volumes/" + volume;
                 #else
                 location = volume;
                 #endif
-                location += path.replace(QString(":"), QString(""));
+                // Traktor uses /: instead of just / as delimiting character for some reasons
+                location += path.replace("/:", "/");
```
Comment (uklotzde) [inline src/library/traktor/traktorfeature.cpp]: Good finding. The replacement is done twice (see below), so moving it into an inlined function (including the comment) would be even better.

**Comment by uklotzde** [comment]
Good finding :+1: The code before was actually buggy and tried to fix the shortcomings with an ugly workaround.

**Comment by daschuer** [comment]
Done. Thank you for review.

**Comment by uklotzde** [comment]
LGTM. Thank you.

**Pull request merged**